lms/djangoapps/instructor_task/tasks.py

```diff
@@ -214,6 +215,18 @@ def exec_summary_report_csv(entry_id, xmodule_instance_args):
 
 
 @task(base=BaseInstructorTask, routing_key=settings.GRADES_DOWNLOAD_ROUTING_KEY)  # pylint: disable=not-callable
+def proctored_exam_results_csv(entry_id, xmodule_instance_args):
+    """
+    Compute proctored exam results report for a course and upload the
+    CSV to an S3 bucket for download.
```

Comment:
We don't really know (or care) where the report is being stored at this point in the call tree -- I'd remove the bit about the S3 bucket